packages/docs/guide/recipes/authorization.md

```
@@ -35,6 +35,12 @@ app.get('/client-permissions', async (req, res) => {
 
 Hooks run against the **target schema** of each path segment — `include=client.realm` checks `client` on this schema and `realm` on the client schema — so an include can never widen access past the related schema's own gate. Rejections follow the schema failure policy: dropped by default, thrown (`ErrorCode.KEY_VALIDATE_REJECTED`) with `throwOnFailure`. An absent context reaches the hook as `undefined`, so a permission hook written like the one above fails closed.
 
+The `relations` hook is not scoped to `include=`. A dotted `filter[client.name]`, `fields[client]` or `sort=client.name` forces the same join, so the hook runs for those too — evaluated once per distinct relation across the query. An actor denied `client` cannot slip the join in through another parameter.
+
+::: warning Authorization happens at the parse/decode boundary
+The gate runs while **parsing untrusted input** — `parse()` / `decode()` and the per-parameter `decode*` helpers. It is a property of that boundary, not of the `Query` object itself: a query you assemble server-side with the schema-free [build layer](/guide/building-queries) (`defineQuery`, condition helpers) and hand straight to an adapter is **not** re-checked — the adapter joins what the query names. Feed client input through a parser/codec with a schema and `context`; keep server-authored queries (and schema `default`s) as the trusted baseline. See [Scoping](#scoping-inject-conditions-the-client-can-t-displace) for pinning server conditions the client can't displace.
```

markdownlint-cli2 (0.23.0)

[warning] Link fragments should be valid

(MD051, link-fragments)